TriblerGUI/test/widgets/trustpage/js/test_radial_positioning.js

```diff
@@ -2,19 +2,21 @@
  * This file tests the graph_to_tree.js file with unit tests
  */
 assert = require('assert');
-graph_to_tree = require('../../../../widgets/trustpage/js/graph_to_tree.js');
+RadialPositioning = require('../../../../widgets/trustpage/js/radial_view/radial_positioning.js');
```

Comment:
@yinghaodai Did we not have another way of requiring dependencies without the relative path?